modules/services/pulseeffects.nix

```diff
@@ -16,10 +16,9 @@ in {
 
     package = mkOption {
       type = types.package;
-      default = pkgs.pulseeffects;
```

Comment:
These changes seem unrelated to the easyeffects module? If so, please do them in a separate commit.